components/micro-gateway-core/src/main/java/org/wso2/micro/gateway/core/Constants.java

```diff
@@ -58,7 +56,6 @@
     public static final String JSON_CONTENT = ".application/json.schema.$ref";
     public static final String SCHEMA = ".schema";
     public static final String EMPTY_ARRAY = "[]";
-    public static final String INTERNAL_ERROR_CODE = "500";
```

Comment:
Isn't this constant have usage in https://github.com/wso2/product-microgateway/blob/master/components/micro-gateway-core/src/main/ballerina/src/gateway/filters/throttle_filter.bal#L186

Reply:
It uses this constant 
https://github.com/wso2/product-microgateway/blob/master/components/micro-gateway-core/src/main/ballerina/src/gateway/constants/api_throttle_out_constants.bal#L21